### DISCUSSION TITLE
Feature Request: Create Entry Link

### DISCUSSION THREAD

**Discussion by ar-ml** [keepassxreboot/keepassxc#11527] [Ideas]
If I rightclick on a password entry I can clone it.
It would be great to also be able to create a "Link" to the password entry! 
(Similar to Windows file links.)

The link then can be moved to another folder. Whenever the link is opened, the original entry would be opened.

Usecase Example:
An admin wants to share credentials for login.example.com with a user.  Therefore he will create a new folder "john.doe" and  share this folder with his colleague via KeeShare. But the admin also has another folder with the name "example.com" and there he wants to have a list of all credentials that he  created for this website. It would be great to have the possibility to create links to the original password entry, to avoid duplicates. 

I would be willing to donate if someone could implement this feature. Cheers!

**Comment by droidmonkey** [comment]
Links are not supported by kdbx specification, but references are. You can convert the username and password fields to references when you clone, or craft your own: https://keepassxc.org/docs/KeePassXC_UserGuide#_entry_cross_reference

**Comment by ar-ml** [comment]
thx for your fast reply @droidmonkey 

I just discovered the "Replace username and password with references check-box" when cloning entries. I understand and like this approach, but from my perspective it would be even more useful to also automatically create references for title, URL and Notes. Would it be possible to change that behaviour?

![image](https://github.com/user-attachments/assets/b60c3c7d-6b8f-41ea-ade4-829bf1e3c94b)

**Reply by ar-ml** [reply]
Additionally, from a UX perspective, it might be better to have an option "Clone with References" directly in the context menu (because this is where I would search for the possibility to create links) - but that is only "nice to have".

![image](https://github.com/user-attachments/assets/112c2029-345a-4c47-929e-e350a7f8ed3d)

**Comment by michaelk83** [comment]
> But the admin also has another folder with the name "example.com" and there he wants to have a list of all credentials that he created for this website.

You can use a tag instead of a folder.
